documents/helpers/coding_standard.md

```diff
@@ -0,0 +1,30 @@
+# C Coding Standard
+
+We will use code formatting standards outlined [here](https://users.ece.cmu.edu/~eno/coding/CCodingStandard.html#brace).
```

Comment:
Why establish your standard formatting around C if your stack is going to be more or less entirely C++?

Reply:
My initial idea was to write C-esque code in C++ because I am used to using C for embedded projects. I did some research, and it seems like making the switch to object oriented embedded C++ is a good idea. I am re-writing the code to be object oriented. However, the addon for cppcheck is only MISRA C 2012. And it doesn't look like there is support for C++ embedded coding guidelines. Ideally, I would like to use AUTOSAR C++ (which is an addendum to the MISRA C++ guidelines), but it looks like I have to pay to use the static analysis tools.

So it looks like cppcheck is the best we are gonna get for C++ static analysis. However, I can link to the AUTOSAR guidelines as a style guide.